dashboard/src/modules/components/FooterComponent/index.less

```diff
@@ -0,0 +1,13 @@
+.footer-container {
+    width: 100%;
+    padding: 1.5rem;
+    background: #151515;
+    color: #ffffff;
+    font-size: .85em;
+    overflow-x: hidden;
+    text-align: center;
+
+    span {
+        font-style: italic;
+    }
+}
```

Comment:
A trailing newline would be nice, if only to avoid that annoying GitHub red circle!

Reply:
Done